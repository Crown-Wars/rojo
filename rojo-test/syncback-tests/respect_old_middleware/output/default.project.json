{
  "name": "respect_old_middleware",
  "tree": {
    "$className": "DataModel",
    "Workspace": {
      "$path": "Workspace"
    }
  },
  "syncbackRules": {
    "ignoreReferents": true
  }
}