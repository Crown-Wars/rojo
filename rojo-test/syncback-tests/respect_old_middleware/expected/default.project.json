{
  "name": "respect_old_middleware",
  "tree": {
    "$className": "DataModel",
    "Workspace": {
      "$properties": {
        "CSGAsyncDynamicCollision": {
          "Enum": 0
        },
        "DecreaseMinimumPartDensityMode": {
          "Enum": 0
        },
        "StreamingEnabled": true
      },
      "$path": "Workspace"
    }
  },
  "syncbackRules": {
    "ignoreReferents": true
  }
}